pkg/build/api/v1/types.go

```diff
@@ -620,6 +620,9 @@ type GenericWebHookEvent struct {
 
 	// Git is the git information if the Type is BuildSourceGit
 	Git *GitInfo `json:"git,omitempty"`
+
+	// Env contains additional environment variables you want to pass into a builder container
+	Env []kapi.EnvVar `json:"env,omitempty"`
```

Comment:
> @bparees - just to be sure, are you referring to AllowEnv being a boolean toggle we would add to https://github.com/openshift/origin/blob/master/pkg/build/api/types.go#L573-L576 ?

that was my thinking, yes, although that struct is shared between generic and git webhook definitions, so it would mean the AllowEnv field would have no meaning when users were defining github webhooks.  not ideal.